src/ipc/probe_support.c

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Artur Kloniecki <arturx.kloniecki@linux.intel.com>
+
+#include <ipc/info.h>
+#include <config.h>
+
+const struct sof_ipc_probe_support probe_support = {
+	.ext_hdr = {
+		.hdr.cmd = SOF_IPC_FW_READY,
+		.hdr.size = sizeof(struct sof_ipc_probe_support),
+		.type = SOF_IPC_EXT_PROBE_INFO,
+	},
+#if CONFIG_PROBE
+	.probe_points_max = CONFIG_PROBE_POINTS_MAX,
+	.injection_dmas_max = CONFIG_PROBE_DMA_MAX
+#else
```

Comment:
The else is not needed sinec they will be zero by default.

Reply:
done